libraries/AP_Rally/AP_Rally.h

```diff
@@ -52,6 +52,10 @@ class AP_Rally {
         }
         return (uint8_t)ret;
     }
+    void set_rally_total(uint8_t num) {
```

Comment:
It's called `truncate`.

Reply:
Maybe mimic the API to bring truncate in (with the range checking we had previously discussed)? I'm basically trying to prevent people from outside the library enabling corrupted rally points. (This is how you get the weird results like what I sent you from MAVProxy the other day)